recipes/sdformat/bld.bat

```diff
@@ -0,0 +1,32 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "NMake Makefiles" ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=ON ^
+    -DBOOST_ROOT=%LIBRARY_PREFIX% ^
+    -DBoost_NO_SYSTEM_PATHS=ON ^
+    -DBoost_NO_BOOST_CMAKE=ON ^
+    -DBoost_DEBUG=ON ^
+    -DUSE_EXTERNAL_TINYXML=ON ^
+    -DUSE_INTERNAL_URDF=OFF ^
+    -DURDF_FOUND=ON ^
```

Comment:
I created a TODO for `urdfdom-feedstock`: https://github.com/conda-forge/urdfdom-feedstock/issues/2 :)